pelican/tools/templates/tasks.py.jinja2

```diff
@@ -147,3 +148,7 @@ def gh_pages(c):
           '-m {commit_message} '
           '{deploy_path} -p'.format(**CONFIG))
 {% endif %}
+
+def pelican_run(cmd):
+    cmd += ' ' + program.core.remainder  # allows to pass-through args to pelican
+    pelican_main(cmd.split(' '))
```

Comment:
`pelican_main` isn't defined anywhere?

Reply:
another potential issue: I'm not sure this will behave well with arguments containing spaces (e.g. paths).

Reply:
Well spotted !
Thanks, I fixed both things